README.md

```diff
@@ -1,6 +1,48 @@
-# Reactive Faucet Deployment Instructions
+# Reactive Faucet Demo
```

Comment:
Please change the wording, it's not just a "demo", but an actual app (arguably, the first real reactive app) we're using.

Reply:
done